core/src/main/java/com/onelogin/saml2/util/Util.java

```diff
@@ -477,17 +479,16 @@ private static String chunkString(String str, int chunkSize) {
 	 *
 	 * @return Loaded Certificate. X509Certificate object
 	 *
-	 * @throws UnsupportedEncodingException 
 	 * @throws CertificateException 
 	 *
 	 */
-	public static X509Certificate loadCert(String certString) throws CertificateException, UnsupportedEncodingException {
+	public static X509Certificate loadCert(String certString) throws CertificateException {
 		certString = formatCert(certString, true);
 		X509Certificate cert;
 		
 		try {
 			cert = (X509Certificate) CertificateFactory.getInstance("X.509").generateCertificate(
-				new ByteArrayInputStream(certString.getBytes("utf-8")));
+				new ByteArrayInputStream(certString.getBytes(StandardCharsets.UTF_8)));
```

Comment:
So using StandardCharsets.UTF_8, UnsupportedEncodingException is not raised anymore?
We avoided the use of StandardCharsets since it requires Java 1.7 and this toolkit in its origen was designed to work with 1.6  but we already introduced 1.7 dependencies.. so its ok.

Reply:
Good to know. What about Java8?

Reply:
as you can read on the readme.
Version 2.X.X, compatible with java7 / java8.

Reply:
Sure! I completely zap this part. :D